### PR TITLE
capcut 3.3.0.1159 (new cask)

### PR DESCRIPTION
**Pull request by rternavskyi**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Review by razvanazamfirei — COMMENTED** [review]
This looks generally good, but I'm not sure how stable the livecheck will be given the `device_id` in it.

**Review by samford — COMMENTED** [review]
> I'm not sure how stable the livecheck will be given the `device_id` in it.

It looks like the relevant part of the JSON remains the same if we remove the `device_id` parameter. The same is true if we remove `arch_info=arm64`.

If these parameters were simply what the application was using when checking for new versions, then we should adapt them to what's appropriate for the cask and minimize them as much as possible. Removing any of the other parameters causes the check to fail, so I think we just want to remove `device_id` and maybe `arch_info` (unless there's a reason for us to restrict this to `arm64` only).

**Comment by bevanjkay** [comment]
I out in considerable work to get the livecheck block working last time, but never heard back from @rternavskyi about whether the corresponding installation changes had any side effects. 

Ref: https://github.com/Homebrew/homebrew-cask/pull/162456

File: Casks/c/capcut.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+cask "capcut" do
+  version "3.3.0.1159"
+  sha256 "26c253719a61eae679a2923121a83d0a39b98d920a5c57f3e00d66224986c867"
+
+  url "https://lf16-capcut.faceulv.com/obj/capcutpc-packages-us/packages/CapCut_#{version.dots_to_underscores}_capcutpc_0_creatortool.dmg",
+      verified: "lf16-capcut.faceulv.com/obj/capcutpc-packages-us/packages/"
+  name "CapCut"
+  desc "Creative platform powered by AI that enables video editing and image design"
```
Comment (bevanjkay) [inline Casks/c/capcut.rb:8]:
```
  desc "Video editing and image design platform"

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Comment by rternavskyi** [comment]
@bevanjkay I'm sorry, last time I didn't have the opportunity to go to github and check the pull request update. So now I'm planning to close it.
I checked that the application is installed correctly. But after logging into the app, it offered me to update the version. It looks like there is another source of the update.

**Comment by bevanjkay** [comment]
@rternavskyi Sometimes apps provide partial updates, that aren't offered as a full reinstall, so it could be expected behaviour

**Comment by rternavskyi** [comment]
> Sometimes apps provide partial updates, that aren't offered as a full reinstall, so it could be expected behaviour

Then everything works perfectly

**Comment by rternavskyi** [comment]
But I don't know if I should delete arch_info and device_id?

**Comment by bevanjkay** [comment]
> But I don't know if I should delete arch_info and device_id?

Yes let's remove them. Thanks.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**